### ISSUE TITLE
Create a Reactive System for Adding consideration based on AI State Components 

### ISSUE THREAD

**Issue by DreamersIncStudios** [DreamersIncStudios/ECS-IAUS-sytstem#20]
Gave Up on this based on inability to easily a Dynamic buffer. Review at later date as it is not required

**Comment by DreamersIncStudios** [comment]
Work completed in 0.5.1.1A branch. Waiting on SystemBase Refactor to merge into Main Branch
